components/MaintainerCard/MaintainerCard.tsx

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable @next/next/no-img-element */
+import styles from "./MaintainerCard.module.css"
+
+export interface MaintainerCardProps {
```

Comment:
none of the props need to start with the word 'maintainer'. It is implied from the component.